controls/radchartview/axes/categoricalaxis.md

```diff
@@ -10,21 +10,44 @@ position: 3
 
 # Categorical Axis
 
+The categorical axis displays a range of categories and the values that determine each category do not define an order. Here is an example of a categorical axis with a few data points.

```

Comment:
"...a set of categories..." would be more appropriate in this case. Also, you can split the first sentence into two sentences. Currently, it is not much understandable. For example, "The categorical axis displays a set of categories. The values that determine each category do not define an order". Also, I don't understand the second part of the first sentence - "The values that determine each category do not define an order". Try to re-phrase this.